index.js

```diff
@@ -87,4 +87,4 @@ for (let i = 0; i < teachers.length; i += 1) {
 <div class="gray-line ${teachers[i].class}"></div>
 `;
   container.append(instructor);
```

Comment:
- [ ] Kindly give your cards the `hover effect` like in the original design
 
![image](https://user-images.githubusercontent.com/98453979/219082527-d0d980b8-5b05-4040-8a5c-ded84e7c6a63.png)
### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/219082729-e65505d6-ba0e-4718-93ff-2534666aab29.png)

Reply:
Great work on implementing this change :+1: